doc/grids.rst

```diff
@@ -304,6 +305,103 @@ Xgcm can also deal with complex topologies such as cubed-sphere and
 lat-lon-cap.
 This is described in the :ref:`grid_topology` page.
 
+
+.. _Boundary conditions:
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~
+
+For variables located at the boundaries, some operations need boundary conditions.
+Let's use the previous example axis, with center and left points::
+
+
+    |     |     |     |     |
+    U--T--U--T--U--T--U--T--|
+    |     |     |     |     |
+
+
+If we have a variable at the U (left) points, we have a problem for some operation
+(e.g. differentiating): how to treat the last T point?
+The solution is to add an extra point for the computation ('X' point on the following sketch)::
+
+
+    |     |     |     |     |
+    U--T--U--T--U--T--U--T--X
+    |     |     |     |     |
+
+
+Different options are possible (``fill`` this extra values with a certain number,
+``extend`` to the nearest value, ``extrapolate`` linearly using the 2 nearest points,
+or periodic condition if the grid axis is periodic).
+Attention, this boundary condition is used to give the value of X, not to give the value of the 
+boundary T point after the operation.
+
+We can illustrate it by creating some data located at the U point:
+
+.. ipython:: python
+
+    g = xr.ufuncs.sqrt(ds.x_g + 0.5) + xr.ufuncs.sin((ds.x_g - 0.5) * 2 * np.pi / 8)
```

Comment:
```suggestion
    g = np.sqrt(ds.x_g + 0.5) + np.sin((ds.x_g - 0.5) * 2 * np.pi / 8)
```

You should never have to explicitly call `xr.ufuncs`.

Reply:
Oh okay I did not know that, thanks!